src/System.IO/src/System/IO/Stream.cs

```diff
@@ -28,6 +28,36 @@ internal SemaphoreSlim EnsureAsyncActiveSemaphoreInitialized()
             // WaitHandle, we don't need to worry about Disposing it.
             return LazyInitializer.EnsureInitialized(ref _asyncActiveSemaphore, () => new SemaphoreSlim(1, 1));
         }
+        
+        // This method is used by CopyTo, CopyToAsync,
+        // and NullStream's override of the latter.
+        internal void ValidateCopyToArguments(Stream destination, int bufferSize)
```

Comment:
I did it since the previous method is the only internal/private method in the class, and it's defined at the top of the file.